### DISCUSSION TITLE
Shared Folder in Airflow

### DISCUSSION THREAD

**Discussion by alexypuli** [apache/airflow#28554] [General]
Hi,
I have mistakenly changed the ownership of `/shared` folder using `chown airflow:airflow -R /shared/` After this operation the Airflow is showing error as `[DAG Import Errors (10)])`. I reverted the ownership of folders inside `/shared,` but still the error is available. Since I have changed ownership with `-R` all the folder and files inside the `/shared` directory have airflow permission. Is it possible to revert back the permission or what is the correct permission we need to provide for the files and folder in `/shared` directory.
Any help on this is much appreciated

**Comment by potiuk** [comment]
We don't know what your "shared" folder is - but generally maybe install airflow again in the same way you did before on a separate machine and check what permissions you have there.

**Comment by potiuk** [comment]
This is the easiest way to rectify the mistake, I think,
